can_tools/scrapers/official/PA/pa_vaccines.py

```diff
@@ -124,7 +124,7 @@ def normalize(self, resjson):
 
         # Dump records into a DataFrame
         df = pd.DataFrame.from_records(data_rows).dropna()
-        df = df.query("location_name != '' & location_name != 'Out-of-State'")
+        df = df.query("location_name != '' & location_name != 'Out-of-State*'")
```

Comment:
Just to confirm, we don't have to worry about the other location names that the demographics scraper does?  https://github.com/covid-projections/can-scrapers/pull/252/files#diff-110778fd367ad022afb439a15950c6a6b7257e77cf95c7d2d86dcd296956bcd0R194

Reply:
Correct!